### ISSUE TITLE
Unable to add Log4j log

### ISSUE THREAD

**Issue by guneetgarg** [anshooarora/extentreports#143]
Am unable to add log4j log in extent report. while running am not getting any error

So How to add log4j log while using TestNG Listener

**Comment by anshooarora** [comment]
See this example:  https://github.com/anshooarora/extentreports/blob/master/java/extentreports/src/test/java/com/relevantcodes/extentreports/testngexamples/ExtentReporterNG.java

The code will add your TestNg logs to the report:

```java
for (String s : Reporter.getOutput()) {
    extent.setTestRunnerOutput(s);
}```

**Comment by guneetgarg** [comment]
i have added above code earlier too but am only seeing pass and fail in report

am using testng listener only with log4j

**Comment by anshooarora** [comment]
The logs are sent to the report from line 63 of the above link.  If you want more details, you will have to use Extent as a logger instead of the using the listener which runs when all the tests have completed running.

**Comment by guneetgarg** [comment]
ok thanks
When we can see new version for java

**Comment by anshooarora** [comment]
Hopefully next week.

**Comment by guneetgarg** [comment]
for (String s : Reporter.getOutput()) {
extent.setTestRunnerOutput(s);
}

**Comment by anshooarora** [comment]
Merged with #151

**Comment by RahulVermaAutomation** [comment]
Hi Anshoo,

I am using Extent report for the reporting purpose as of now. But now I want it to use Log4j logs and display it in the reports. I am using Log4j for the logging and dont want to add any other code for logging on my scripts. I am using the same code which you have given above in github. Could you please provide me code of integrating log4j with extent reports.

Thanks,
Rahul
